packages/oruga/src/components/autocomplete/Autocomplete.vue

```diff
@@ -351,14 +351,12 @@ function setItemRef(
 const { checkHtml5Validity, onInvalid, onFocus, onBlur, isFocused, setFocus } =
     useInputHandler(inputRef, emits, props);
 
-const vmodel = useVModelBinding<string | number>(props, emits, {
-    passive: true,
-});
+const vmodel = defineModel<string | number>({ default: undefined });
```

Comment:
I didn't know about generic components, that's great! So, if we make the suggested change, do you think it will work? Right now, I think we're missing a type.

```suggestion
const vmodel = defineModel<T>({ default: undefined });
```

Reply:
No, that was a misunderstanding I had at first too. The vmodel here is not the selected option. The vmodel is the input value, which can be an option or a property, defined by the field prop, of an option. So the vmodel is a string because the input value is a string (I think the o-input does not work for number, even a number type is acceptable, but that is a topic for later).

Currently the selected option is only emitted by the `select` event, and doesn't have a two-way binding. 
Maybe we should change that? But this will be a breaking change and will only be introduced when we make the 0.9 release.

Reply:
Ah, got it. Thanks for the explanation.